client/src/routes.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Switch, Route } from "react-router-dom";
+import HomePage from "./containers/HomePage";
+import AddressPage from "./containers/AddressPage";
+import BBLPage from "./containers/BBLPage";
+import AboutPage from "./containers/AboutPage";
+import HowToUsePage from "./containers/HowToUsePage";
+import MethodologyPage from "./containers/Methodology";
+import TermsOfUsePage from "./containers/TermsOfUsePage";
+import PrivacyPolicyPage from "./containers/PrivacyPolicyPage";
+
+export const localizeRoute = (route: string) => "/:locale" + route;
+
+export const createRouteForAddressPage = (boro: string, streetname: string, housenumber?: string) =>
+  `/address/${boro}/${housenumber || " "}/${streetname}`;
+
+const addressPageRouteWithParams = localizeRoute(
+  createRouteForAddressPage(":boro", ":streetname", ":housenumber")
+);
+
+export const WhoOwnsWhatRoutes = () => (
+  <Switch>
+    <Route exact path={localizeRoute("/")} component={HomePage} />
+    <Route
+      path={addressPageRouteWithParams}
+      render={(props) => <AddressPage currentTab={0} {...props} />}
+      exact
+    />
+    <Route
+      path={addressPageRouteWithParams + "/timeline"}
+      render={(props) => <AddressPage currentTab={1} {...props} />}
+    />
+    <Route
+      path={addressPageRouteWithParams + "/portfolio"}
+      render={(props) => <AddressPage currentTab={2} {...props} />}
+    />
+    <Route
+      path={addressPageRouteWithParams + "/summary"}
+      render={(props) => <AddressPage currentTab={3} {...props} />}
+    />
+    <Route path={localizeRoute("/bbl/:boro/:block/:lot")} component={BBLPage} />
+    <Route path={localizeRoute("/bbl/:bbl")} component={BBLPage} />
```

Comment:
Since these parametrized routes are not actually generated anywhere in the code (for a Redirect or whatnot), I didn't bother making their own `createRoute...` function like I did with the Address Page Routes